hydra/utils.py

```diff
@@ -133,5 +162,25 @@ def _instantiate_class(
 
     for k, v in rest.items():
         final_kwargs[k] = v
+    return final_kwargs
 
+
+def _instantiate_class(
+    clazz: Type[Any], config: PluginConf, *args: Any, **kwargs: Any
+) -> Any:
+    final_kwargs = _get_kwargs(config, **kwargs)
     return clazz(*args, **final_kwargs)
+
+
+def _call_callable(
+    fn: Callable[..., Any], config: PluginConf, *args: Any, **kwargs: Any
+) -> Any:
+    final_kwargs = _get_kwargs(config, **kwargs)
+    return fn(*args, **final_kwargs)
+
+
+# aliases
+instantiate = call
+get_static_method = get_fn_or_method
+get_method = get_fn_or_method
+_get_class_name = _get_cls_name
```

Comment:
There is only one usage of `_get_class_name()` in Hydra outside of this class. just fix it. no need to introduce an alias.

Reply:
:+1: